functions/export-csv.js

```diff
@@ -21,13 +21,61 @@ const {db} = require('./common/context');
 
 function exportCsv(req, res) {
   const {
-    project: providedProjectId,
-    featureType: providedFeatureTypeId,
+    project: projectId,
+    featureType: featureTypeId,
     lang: desiredLanguage,
   } = req.query;
 
-  return res.status(500).send('Not supported yet.');
+  let data = {};
+  return db.fetchProject(projectId).then(
+    project => {
+      if (!project.exists) {
+        res.status(404).send('not found');
+        return Promise.reject(new Error('project not found: ' + projectId))
+      }
+      res.type('text/csv');
+      let form = project.get('featureTypes')[featureTypeId]['forms'];
+      let elements = form[Object.keys(form)[0]]['elements'];
+      var arr = []
+      for (var el in elements) {
+        arr.push('"' + escapeQuotes(elements[el]['labels']['_']) + '"');
+      }
+      res.write(arr.join() + '\n');
```

Comment:
Did you mean `arr.join(',')`? How do the commas get added otherwise?

Reply:
By default they're joined separated by comma:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/join

Reply:
Ah thanks, I didn't realize. May be worth making it explicit anyway (but urgent).

Reply:
Changed to arr.join(',')